### PR TITLE
Add some basic autofocus tests

### PR DESCRIPTION
**Pull request by domenic**
Follows https://github.com/whatwg/html/pull/3488, although certainly doesn't test the underlying issue exhaustively.

Edge, Firefox, and Chrome pass all these. Haven't tested Safari.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25238)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/349181358)

File: html/semantics/forms/autofocus/first-when-later.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<script src="/resources/testharnessreport.js"></script>
+
+<input autofocus>
+
+<script>
+"use strict";
+
+const input1 = document.querySelector("input");
+const input2 = document.createElement("input");
+input2.autofocus = true;
```
Comment (annevk) [inline html/semantics/forms/autofocus/first-when-later.html]: You're not inserting the element. It still get focus?

**Review by annevk — COMMENTED** [review]
first-when-later.html looks incorrect, otherwise LGTM.

File: html/semantics/forms/autofocus/first-when-later.html (hunk 1 above)
Comment (domenic) [reply html/semantics/forms/autofocus/first-when-later.html]: Well, the test is that input1 keeps the focus, so the test still passes. But yeah, this is a bug that makes the pass trivial. Fixing.

**Review by annevk — APPROVED** [review]
It'd be a little more compelling I think if you inserted it before the other input element, but this seems fine as initial tests.

**Comment by domenic** [comment]
Oh, good idea, Chrome fails that. I'll add it as a separate file and file a bug.

**Review by annevk — APPROVED** [review]

**Pull request merged**